src/Package/Impl/Microsoft.VisualStudio.R.Package.csproj

```diff
@@ -1602,113 +1601,33 @@
       <VSIXSourceItem Include="$(OutputPath)\Broker\Windows\Microsoft.Net.Http.Headers.dll" />
       <VSIXSourceItem Include="$(OutputPath)\Broker\Windows\Microsoft.Net.Http.Server.dll" />
       <VSIXSourceItem Include="$(OutputPath)\Broker\Windows\Microsoft.Win32.Registry.dll" />
-      <VSIXSourceItem Include="$(OutputPath)\Broker\Windows\Microsoft.Win32.Primitives.dll" />
```

Comment:
This may need to be reflected in setup (remote msi and VS swix)

Reply:
Done